runtime/lua/vim/_system.lua

```diff
@@ -79,7 +79,7 @@ function SystemObj:_timeout(signal)
   self:kill(signal or SIG.TERM)
 end
 
-local MAX_TIMEOUT = 2 ^ 31
+local MAX_TIMEOUT = 2 ^ 31 - 1
```

Comment:
```suggestion
-- Use max 32-bit signed int value to avoid overflow on 32-bit systems. #31633
local MAX_TIMEOUT = 2 ^ 31 - 1
```

Reply:
Should I use a particular message for the commit? I see you have conventions, but while I could figure out that the type of this commit is `fix`, I have no idea what the scope is.

Or is it possible that you update the commit message during the merge process to conform to the convention?

Reply:
will fix it during merge